doc/reactions.md

```diff
@@ -8,166 +8,75 @@ Reactions allow SQL LRS to watch for patterns in submitted xAPI data and dynamic
-
-Alternative approaches to Identity Path may be used by modifying `identityPaths`, for instance `["context", "registration"]` to group statements by learning session.
+![reaction edit](images/reactions/edit_intro.png)
+Creating a new reaction will display an edit page, where the user can modify the reaction title, active status, and ruleset.
 
 ### Conditions
 
-`conditions` is a mapping of names to rules for finding significant statements. Rules can be composed with boolean logic.
+Each condition is a set of rules for finding significant statements. Each condition has a unique name followed by its rules, which can be composed with boolean logic.
 
-In the example given above statement `a` must have an object id equal to `https://example.com/activities/a`, a verb id equal to `https://example.com/verbs/completed`, and a result success equal to `true`. Statement `b` must have the same verb and result success but an object id equal to `https://example.com/activities/b` and a timestamp greater than that of `a`.
+![reaction edit alpha](images/reactions/edit_condition_alpha.png)
+In the example given, statement "alpha" must have an object `id` equal to `https://example.com/activities/alpha`, a verb `id` equal to `https://example.com/verbs/completed`, and its result `success` property equal to `true`.
+
+![reaction edit beta](images/reactions/edit_condition_beta.png)
+![reaction edit beta ref](images/reactions/edit_condition_beta_ref.png)
+Statement "beta" must have the same verb and result success but an object `id` equal to `https://example.com/activities/beta` and a timestamp greater than that of statement "alpha."
 
 #### Rules
 
-All rules have a `path` array that indicates a path in an xAPI statement and an `op` that is one of the following operators:
+All rules have a path array that indicates a path in an xAPI statement and an operator that is one of the following:
 
-* `gt` - Greater than
-* `lt` - Less than
-* `gte` - Greater than or equal
-* `lte` - Less than or equal
-* `eq` - Equal
-* `noteq` - Not equal
-* `like` - Fuzzy match using SQL `%` syntax. For example, `bo%` matches `bob` and `boz`.
-* `contains` - Array contains
+* Greater than
+* Less than
+* Greater than or equal
+* Less than or equal
+* Equal
+* Not equal
+* Like (fuzzy match using SQL `%` syntax; for example, `bo%` matches `bob` and `boz`.)
+* Array contains
 
 Rules either have a `val` literal value or a `ref` which is a path into a statement found for another condition.
 
 #### Booleans
 
 Booleans compose multiple rules together. Booleans are objects with a single key:
 
-* `and` - Array of rules which must all be true
-* `or` - Array of rules of which one must be true
-* `not` - Rule to nullify
+* AND: Array of rules which must all be true
+* OR: Array of rules of which one must be true
+* NOT: Rule that must _not_ be true
+
+Rule types (either "Statement Criteria" or a boolean) can be selected using the topmost select input, or via the "Add sub-clause" button.
```

Comment:
insert a section after boolean about _nesting_ and how you can go as deep as you need to.

Reply:
https://github.com/yetanalytics/lrsql/pull/428/commits/fe4b91eb7fce62546ccc15c6b41c2a59097b3108